docs/setup.md

```diff
@@ -10,23 +10,25 @@
-| ------------------------------------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| Title                                      | Specify a title for the SOC Overview Sankey.                                                                                                                                                      |
-| Resource                                   | Select the module.                                                                                                                                                       |
-| Filter Criteria                            | Define the filter criteria using which to filter the data retrieved by this widget.                                                                                                                                             |
-| Source Nodes                               | Select the field to render source nodes.                                                                                                                                                               |
-| Target Nodes                               | Select the picklist to render target nodes.                                                                                                                                                    |
-| Relationship Resource                      | Select the relationship module to get picklist details for relationship nodes.                                                                                                                                                                        |
-| Relationship Nodes                         | Select the picklist to render relationship nodes.                                                          |
-| External Source JSON                       | Enter JSON data to render sources which generates an events.                                                                                       |
+A Sankey chart needs a source node and a target node. It then represents the flow from the source node to the target node using paths known as links. Following table helps customize the **SOC Overview Sankey** widget:
+
+| Fields                | Description                                                                         |
+|-----------------------|-------------------------------------------------------------------------------------|
+| Title                 | Specify a title for the SOC Overview Sankey widget as it appears on the dashboard.  |
+| Resource              | Select the module whose Sankey representation is to be displayed.                   |
```

Comment:
whose Sankey representation is to be displayed, i.e. FortiSOAR module

Reply:
Fixed.